contracts/extensions/Bank.sol

```diff
@@ -150,6 +152,32 @@ contract BankExtension is DaoConstants, AdapterGuard, IExtension {
         return availableTokens[token];
     }
 
+    /**
+     * @notice Sets the maximum amount of internal tokens allowed in the bank
+     * @param maxTokens The maximum amount of token allowed
+     */
+    function setMaxInternalTokens(uint8 maxTokens) external {
```

Comment:
why do we need to limit the number of internal tokens ? I understand for the external ones but I'm unsure why we need this for the internal ones